services/gateway/endpoints/src/test/java/org/eclipse/ditto/services/gateway/endpoints/routes/cloudevents/CloudEventsTest.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2020 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+package org.eclipse.ditto.services.gateway.endpoints.routes.cloudevents;
+
+import org.eclipse.ditto.services.gateway.endpoints.EndpointTestBase;
+import org.eclipse.ditto.services.gateway.util.config.endpoints.CloudEventsConfig;
+import org.junit.Test;
+
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public final class CloudEventsTest extends EndpointTestBase {
```

Comment:
What we normally add to our endpoint tests can be seen e.g. in: https://github.com/eclipse/ditto/blob/master/services/gateway/endpoints/src/test/java/org/eclipse/ditto/services/gateway/endpoints/routes/things/ThingsRouteTest.java

* build the route
* wrap it in Akka HTTP testkit's `testRoute` (as seen in `setUp()`)
* run `POST ` requests against the endpoint providing `ce-*` headers or omitting them in order to test for the exception which may be returned

Reply:
I can do that, but would tackle the problem in another PR.